theme_5/scss/_home.scss

```diff
@@ -0,0 +1,52 @@
+
+.tablet-collections {
+  padding: 0 15px;
+  display: flex;
+}
+
+.category-item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.outer-box {
+  width: 200px;
+  height: 270px;
+  display:block;
+  position:relative;
+
+  img {
+    width: 200px;
+    height: auto;
+  }
+
+  .inner-box {
+    display: flex;
```

Comment:
Properties should be ordered align-items, background, display, height, justify-content, left, opacity, padding, position, top, transition, width